apps/rush-lib/src/logic/Git.ts

```diff
@@ -586,4 +532,17 @@ export class Git {
       );
     }
   }
+
+  private _executeGitCommandAndCaptureOutput(
```

Comment:
Does this belong in [rush-lib/src/logic/Git.ts](https://github.com/microsoft/rushstack/blob/master/apps/rush-lib/src/logic/Git.ts)?

Reply:
That's where it currently is.